module/basicfantasyrpg.mjs

```diff
@@ -4,6 +4,8 @@ import { BasicFantasyRPGItem } from './documents/item.mjs';
 // Import sheet classes.
 import { BasicFantasyRPGActorSheet } from './sheets/actor-sheet.mjs';
 import { BasicFantasyRPGItemSheet } from './sheets/item-sheet.mjs';
+import { CharacterSheet } from './sheets/character-sheet.mjs';
```

Comment:
The ActorSheet was for any actor types - both characters and monsters. Is the new sheet for the same or only for Characters?

I'd like the naming to remain consistent with the previous sheets and avoid any potential collisions in future, unless this naming is expected by application v2?